docs/guides/request_storage.mdx

```diff
@@ -0,0 +1,113 @@
+            {RqWithCrawlerExample}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="request_queue_with_crawler_explicit" label="Explicit usage with Crawler" default>
+        <CodeBlock className="language-python">
+            {RqWithCrawlerExplicitExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+## Request list
+
+The <ApiLink to="class/RequestList">`RequestList`</ApiLink> is a simpler, lightweight storage option, used when all URLs to be crawled are known upfront. It represents the list of URLs to crawl that is stored in a crawler run memory (or optionally in default <ApiLink to="class/KeyValueStore">`KeyValueStore`</ApiLink> associated with the run, if specified). The list is used for the crawling of a large number of URLs, when we know all the URLs which should be visited by the crawler and no URLs would be added during the run. The URLs can be provided either in code or parsed from a text file hosted on the web. The <ApiLink to="class/RequestList">`RequestList`</ApiLink> is typically created exclusively for a single crawler run, and its usage must be explicitly specified.
+
+:::warning
+
+The <ApiLink to="class/RequestList">`RequestList`</ApiLink> class is in its early version and is not fully
+implemented. It is currently intended mainly for testing purposes and small-scale projects. The current
+implementation is only in-memory storage and is very limited. It will be (re)implemented in the future.
+For more details, see the GitHub issue [crawlee-python#99](https://github.com/apify/crawlee-python/issues/99).
+For production usage we recommend to use the <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>.
+
+:::
+
+The following code demonstrates the usage of the <ApiLink to="class/RequestList">`RequestList`</ApiLink>:
+
+<Tabs groupId="request_list">
+    <TabItem value="request_list_basic_example" label="Basic usage" default>
+        <CodeBlock className="language-python">
+            {RlBasicExample}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="request_list_with_crawler" label="Usage with Crawler">
+        <CodeBlock className="language-python">
+            {RlWithCrawlerExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+{/*
+
+## Which one to choose?
+
+TODO: write this section, once https://github.com/apify/crawlee-python/issues/99 is resolved
+
+*/}
+
+## Cleaning up the storages
+
+By default, storages are automatically purged before the crawler starts, unless explicitly configured otherwise, for that case, see <ApiLink to="class/Configuration#purge_on_start">`Configuration.purge_on_start`</ApiLink>. This cleanup happens as soon as a storage is accessed, either when you open a storage (e.g., using <ApiLink to="class/RequestQueue#open">`RequestQueue.open`</ApiLink>) or when interacting with a storage through one of the helper methods (e.g., <ApiLink to="class/BasicCrawler#add_requests">`BasicCrawler.add_requests`</ApiLink>, which implicitly opens the request storage).
```

Comment:
Default storages, that is :slightly_smiling_face:

Reply:
Is it true? It opens the configured storage, does not have to be the default one.

Reply:
only default storages are auto-purged